.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug
+about: Report a bug related to kube-prometheus
+labels: kind/bug
+---
+
+<!--
+
+Feel free to ask questions in #prometheus-operator on Kubernetes Slack!
+
+-->
+
+**What happened?**
+
+**Did you expect to see some different?**
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Environment**
+
+* Prometheus Operator version:
```

Comment:
This should probably no longer be prometheus operator version but eventually `kube-prometheus` version, right?